uaclient/entitlements/tests/test_cis.py

```diff
@@ -63,7 +63,7 @@ def fake_platform(key=None):
             mock.call(
                 "/etc/apt/sources.list.d/ubuntu-cis-audit-xenial.list",
                 "http://CIS-AUDIT",
-                "TOKEN",
+                "%s-token" % entitlement.name,
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.